### PR TITLE
Add JetBrains Theme to site

### PR DESCRIPTION
**Pull request by cbracco**
Hey @zenorocha! 

Whipped up a logo/screenshot/content for the JetBrains theme. Used theme colors, and tried to nail a similar style to the other logos. Let me know what you think. 

I attached some screenshots but you can also see it in action if you checkout this branch and run the site locally.

frankenstein

jetbrains

homepage

Thanks!

### CODE REVIEW

**Comment by heinst** [comment]
Great work with the icon! Theres about 3 or 4 unused ones in the assests folder but this is just too good. I  👍  * 💯

**Comment by cbracco** [comment]
@heinst Oh whoops! I didn’t realize that. Glad you like it anyway. 😋

**Comment by zenorocha** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,4 @@
 dist
-node_modules
\ No newline at end of file
+node_modules
+npm-debug.log
+.DS_Store
```
Comment (zenorocha) [inline .gitignore]: This should be ignored in your `.gitignore_global`

**Comment by zenorocha** [comment]
Wow! That's super impressive @cbracco. In fact, we have a serious issue with these icons. The iconset I chose is almost entirely used and I don't know exactly what do to. Would you be interested in designing these icons? What you did with JetBrains was incredible ;D

**Comment by heinst** [comment]
We still have coffin, hat, and pumpkin dark icons left to use, then well be all out 👎

**Comment by cbracco** [comment]
@zenorocha designing additional ones, or redesigning existing ones, or both?

**Comment by zenorocha** [comment]
I guess both. The ones we currently use doesn't really reflect the IDEs.

**Comment by cbracco** [comment]
@zenorocha I’d be down with that! I am not swimming in free time but I can probably knock them out one by one over time.

**Pull request merged**